spec/models/spree/retail/shopify/generate_refund_order_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+Spree.describe Spree::Retail::Shopify::GenerateRefundOrder, type: :model do
+  include_context 'shopify_request'
+
+  let!(:store) { create :store }
```

Comment:
I probably would, yeah. To have full test coverage would require a _ton_ of specs to cover the myriad control paths, and there's a lot of database access so they'd be slow.

But I recognize the magnitude of effort involved in such a refactoring. Do you feel comfortable with the object as-is? Do you think the effort can be safely deferred?